uiowa/uiowa
# Update to stable release of easysvg.

Update EasySVG dependency.

# How to test

Here is the code comparison: https://github.com/kartsims/easysvg/compare/8f4b75cfb950efd242dc889fb965e3765fea1a29...2.4

- On main, sync brand local - `ddev composer install && ddev blt frontend && ddev blt ds --site=brand.uiowa.edu
`
- Re-save https://brand.uiowa.ddev.site/node/966/edit (CLAS was a common testing lockup because of the characters used)
- Download https://brand.uiowa.ddev.site/lockup/966/download
- Checkout easysvg_update and `ddev composer install && ddev drush @brand.local cr`
- Re-save https://brand.uiowa.ddev.site/node/966/edit and download.
- In Illustrator or other image diff software, overlay and compare for differences. You can try doing a github diff, but mine didn't detect changes to commit.
- Additionally fixed an php 8.1 deprecated rtrim issue. There should be no warnings on save.

We were already using the kerning functionality, so the main changes to the module since seem to be related to php 7.4 and clean up.
